### ISSUE TITLE
amélioration : afficher taux de certification de la commune

### ISSUE THREAD

**Issue by zorglubu** [osm-fr/osm-vs-fantoir#364]
Bonjour,
Serait-il difficile d'afficher quelque part dans la page d'une commune ou dans pifomap le taux de certification des adresses fourni par https://adresse.data.gouv.fr/deploiement-bal ?

En effet, je tente de ne plus trop passer de temps sur les communes peu ou non certifiées depuis que j'ai découvert cette info. Mais je dois aller la rechercher sur adresse.data.gouv.fr, ce qui est long ;-)
Merci.

**Comment by benoit-dd** [comment]
Cette info est déjà présente sur la page par département (https://bano.openstreetmap.fr/pifometre/stats_dept.html#), mais l'avoir directement sur la page de la commune serait top!

exemple un peu brut d'intégration en rajoutant dans la page commune la ligne de la commune telle qu'elle est sur la page département (encadré rouge).
ça créé juste quelques redondances avec l'encadré en haut à droite
![image](https://github.com/osm-fr/osm-vs-fantoir/assets/52233014/a61440fc-7aa7-48ee-aba5-83fe15062429)

**Comment by gendy54** [comment]
Disons que juste avoir le taux d'adresses certifiées, ce serait pas mal. Par exemple juste à côté du nom de la commune. Et sans faire trop de redondance.
Et par la même occasion, trouver une couleur pour une certif à 100% du reste.

**Comment by Koreller** [comment]
Effectivement, je trouve l'information assez utile mais difficile à trouver.

Avoir cette information en avant serait une très bonne idée !

**Comment by benoit-dd** [comment]
maintenant qu'il y a une icône pour afficher l'existence d'une BAL, on pourrait l'améliorer pour afficher le pourcentage (code couleur, barre d'avancement?) et/ou pour faciliter l’icône pourrait être cliquable pour aller directement sur la page de la commune (https://adresse.data.gouv.fr/commune/61079)

<img width="706" height="163" alt="Image" src="https://github.com/user-attachments/assets/4e19852e-b759-409a-995d-57912bafbfea" />
